mmv1/products/gkehub2/Fleet.yaml

```diff
@@ -99,6 +99,28 @@ properties:
     name: "defaultClusterConfig"
     description: The default cluster configurations to apply across the fleet.
     properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: "binaryAuthorizationConfig"
+        description: Enable/Disable binary authorization features for the cluster.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: "evaluationMode"
+            description: Mode of operation for binauthz policy evaluation.
+            values:
+              - DISABLED
+              - POLICY_BINDINGS
+          - !ruby/object:Api::Type::Array
+            name: "policyBindings"
+            description: Binauthz policies that apply to this cluster.
+            output: false
```

Comment:
Why output false? output is assumed by to be false by default

Reply:
Done.